docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -457,10 +469,12 @@ For example, GPU run vs CPU run performance comparison or different runs with di
 We can input multiple Spark event logs and this tool can compare environments, executors, Rapids related Spark parameters,
 
 - Compare the durations/versions/gpuMode on or off:
-```
+
 ### A. Information Collected ###
```

Comment:
For "### A. Information Collected", it is actually output of the tool. So I would suggest we keep it in the code block or we can remote this line to avoid confusion.